app/views/layouts/adopter_foster_dashboard.html.erb

```diff
@@ -70,6 +70,14 @@
                           <i class="fe fe-star nav-icon"></i> Adopted Pets
                         <% end %>
                       </li>
+                       <!-- Nav item -->
+                      <% if current_user.has_role?(:fosterer, current_user.organization) %> 
+                        <li class="nav-item">
+                          <%= active_link_to adopter_fosterer_fostered_pets_path, class: "nav-link" do %>
+                            <i class="fe fe-file-text nav-icon"></i> Fosters
```

Comment:
Can we update this to 'Fostered Pets' please

Reply:
Done